src/data.table.h

```diff
@@ -268,6 +268,9 @@ SEXP substitute_call_arg_namesR(SEXP expr, SEXP env);
 //negate.c
 SEXP notchin(SEXP x, SEXP table);
 
+// topn.c
+SEXP topn(SEXP x, SEXP nArg, SEXP ascArg, SEXP naArg, SEXP sortedArg);
```

Comment:
below, we've avoided naming the args here, I think this is a nice way to avoid them going out of sync. At least for consistency we should remove here too.